### PR TITLE
Merge in Prod bug fixes into master

### PR DESCRIPTION
**Pull request by alexwakeman**
Resolves conflicts and ensures master has the fixes needed to work on Sprint tickets.

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseType.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
   {
     "LiveFrom": "16/05/2019",
     "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
-    "UserRole": "caseworker-divorce-solicitor",
+    "UserRole": "caseworker-divorce-superuser",
```
Comment (qzhou-hmcts) [inline definitions/divorce/json/AuthorisationCaseType.json]: ?

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseType.json]: @hosainnet did this change, was a bug fix I believe...

Comment (qzhou-hmcts) [reply definitions/divorce/json/AuthorisationCaseType.json]: this is the file 
``` {
    "LiveFrom": "16/05/2019",
    "CaseTypeID": "DIVORCE",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "CRU"
  },
  {
    "LiveFrom": "16/05/2019",
    "CaseTypeID": "DIVORCE_NOTICE_OF_ACTING",
    "UserRole": "caseworker-divorce-superuser",
    "CRUD": "CRU"```

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseType.json]: https://github.com/hmcts/div-ccd-definitions/pull/153/commits/9f7bede2e57d0842360ac725590670b89ae7d224 yes it's meant to be

Comment (qzhou-hmcts) [reply definitions/divorce/json/AuthorisationCaseType.json]: ahh yup right

**Pull request merged**